source/retryable-writes/tests/unified/handshakeError.yml

```diff
@@ -0,0 +1,69 @@
+description: "retryable writes handshake failures"
+
+schemaVersion: "1.0"
```

Comment:
Per my earlier comment in the retryable reads test, this needs to be bumped.

Reply:
done!